code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -376,14 +376,19 @@
 	else if(istype(I, /obj/item/weapon/melee/energy/blade) && secure)
 		emag_act(INFINITY, user)
 		return
-	else if(QUALITY_PULSING in I.tool_qualities && secure && locked)
+	else if((QUALITY_PULSING in I.tool_qualities) && secure && locked)
 		user.visible_message(
 		SPAN_WARNING("[user] picks in wires of the [src.name] with a multitool"), \
 		SPAN_WARNING("[pick("Picking wires in [src.name] lock", "Hacking [src.name] security systems", "Pulsing in locker controller")].")
 		)
 		if(I.use_tool(user, src, WORKTIME_LONG, QUALITY_PULSING, FAILCHANCE_HARD, required_stat = STAT_MEC))
 			if(hack_stage < hack_require)
-				playsound(loc, 'sound/items/glitch.ogg', 60, 1, -3)
+				var/obj/item/weapon/tool/T = I
+				if (istype(T))
+					if(!(T.silenced))
+						playsound(loc, 'sound/items/glitch.ogg', 60, 1, -3)
+				else
+					playsound(loc, 'sound/items/glitch.ogg', 60, 1, -3)
```

Comment:
Looks like the same audio settings for both silenced and unsilenced
Give the silenced version a volume of 3 and an extrarange of -5